restli-spring-bridge/src/main/java/com/linkedin/restli/server/spring/RestliHttpRequestHandler.java

```diff
@@ -82,12 +86,29 @@ public RestliHttpRequestHandler(RestLiConfig config,
                                   SpringInjectResourceFactory injectResourceFactory,
                                   FilterChain filterChain)
   {
-    RestLiServer restLiServer = new RestLiServer(config, injectResourceFactory);
+    this(config, injectResourceFactory, filterChain, 0);
+  }
+
+  public RestliHttpRequestHandler(RestLiConfig config,
+                                  SpringInjectResourceFactory injectResourceFactory,
+                                  FilterChain filterChain,
+                                  int parseqThreadPoolSize)
+  {
+    Engine parseqEngine = null;
+    if (parseqThreadPoolSize > 0) {
+      final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(parseqThreadPoolSize);
```

Comment:
You should make the scheduler also a constructor parameter so it can be configured using spring. A service might want to customize the executor service used with the server.

Reply:
Thanks for the review, updated.